Tools/dea_tools/bom.py

```diff
@@ -114,10 +131,7 @@ def ui_select_station(stations,
                       center=(-24, 138),
                       **kw):
     """
-    Returns
-    =======
-
-    (gauge_data, station)
+    Create an interactive map for selecting river gauging stations.
```

Comment:
Suggestion: it might be good to specify the output and its data type.

Reply:
Hey @geoscience-aman, I 100% agree... the trouble is that with this function I'm not entirely sure what's being returned. I've gone down a rabithole and ended up here: 
https://github.com/Kirill888/jupyter-ui-poll/blob/develop/jupyter_ui_poll/_poll.py#L308-L309 
But that output doesn't really make sense to me. I think I might have to leave this for now and revisit later on (and probably improve documentation for a bunch of the functions in this module).